sql/_27_banana_qa/issue_16066_BINARY_charset/_02_coercibility/cases/_09_constant_03.sql

```diff
@@ -4,7 +4,7 @@ set names binary;
 -- constant
 select 1,collation('8'),coercibility('8'),collation('¥' collate binary),coercibility('¥' collate binary) from db_class where '8' > '¥' collate utf8_ja_exp limit 1;
 select 1,collation(('8' > '¥') collate binary),coercibility(('8' > '¥') collate binary) from db_class where ('8' > '¥') collate binary limit 1;
-select coercibility(select _binary'a' union select _binary'b');
+select coercibility("select _binary'a' union select _binary'b'");
```

Comment:
Why is it in quotation marks?
This seems to be irrelevant to the concept of this change.

Reply:
If I do not enclose it in double quotation marks, the syntax error is not resolved.

Reply:
Please do not correct the query and change the error code on the answer sheet.

Reply:
ok, I checked it.